controllers/repository_controller.go

```diff
@@ -140,8 +143,34 @@ func (r *RepositoryReconciler) checkInitial(ctx context.Context, logger logr.Log
 	if instanceDeepCopy.Labels == nil {
 		instanceDeepCopy.Labels = make(map[string]string)
 	}
+	url := instanceDeepCopy.Spec.URL + "/api/charts"
```

Comment:
just skip oci type

Reply:
Currently only support chartmuseum

Reply:
> Currently only support chartmuseum

Because now we only support the ChartMuseum, we should skip the URL that starts with `oci://`. When the URL starts with `oci://`, we don't have to get the username and password from the secret, and we don't need to send a request...

Reply:
@zqq454224016